{
    "has_execution_error": true,
    "execution_output": "Fatal Python error: init_sys_streams: can't initialize sys standard streams\nPython runtime state: core initialized\nOSError: [Errno 9] Bad file descriptor\n\nCurrent thread 0x00000001dcac5c40 (most recent call first):\n  <no Python frame>\n",
    "pred": {
        "error": "malformed_output"
    },
    "gold": {
        "itinerary": [
            {
                "action": "meet",
                "person": "William",
                "start_time": "10:45",
                "end_time": "11:30"
            },
            {
                "action": "meet",
                "person": "Sandra",
                "start_time": "11:40",
                "end_time": "11:55"
            },
            {
                "action": "meet",
                "person": "Elizabeth",
                "start_time": "12:15",
                "end_time": "14:00"
            },
            {
                "action": "meet",
                "person": "Carol",
                "start_time": "14:09",
                "end_time": "15:09"
            },
            {
                "action": "meet",
                "person": "Anthony",
                "start_time": "15:32",
                "end_time": "16:47"
            },
            {
                "action": "meet",
                "person": "Stephanie",
                "start_time": "16:54",
                "end_time": "18:09"
            },
            {
                "action": "meet",
                "person": "Barbara",
                "start_time": "19:15",
                "end_time": "20:30"
            },
            {
                "action": "meet",
                "person": "Kenneth",
                "start_time": "21:15",
                "end_time": "22:00"
            }
        ]
    },
    "status": "Execution error: malformed_output",
    "violated_constraint": {},
    "is_exact_match": false,
    "constraints_satisfied": false,
    "pass_number": 3
}